src/stores/prefs-store.js

```diff
@@ -0,0 +1,96 @@
+import {observable, action} from 'mobx';
+import { findByProperty } from 'util/common';
+import Torrent from './torrent';
+
+export const PrefCookieKeys = {
+  statusFilter: 'filter',
+  sortCriteria: 'sort_method',
+  sortDirection: 'sort_direction',
+  compact: 'compact_display_state'
+}
+
+export const FilterStates = [
+  {value: 0, label: 'All', persistKey: 'all'},
+  {value: 11, label: 'Active', persistKey: 'active'},
+  {value: Torrent.STATUS_DOWNLOAD, label: 'Downloading', persistKey: 'downloading'},
+  {value: Torrent.STATUS_SEED, label: 'Seeding', persistKey: 'seeding'},
+  {value: Torrent.STATUS_STOPPED, label: 'Paused', persistKey: 'paused'},
+  {value: 55, label: 'Finished', persistKey: 'finished'},
+];
+
+class PrefsStore {
+  @observable statusFilter;
+  @observable sortCriteria;
+  @observable sortDirection;
+  @observable compact;
+
+  constructor() {
+    this.rehydrate();
```

Comment:
It'd be better to extract store initialization to a different module, like a bootstrap process or similar.